src/App.jsx

```diff
@@ -1,12 +1,14 @@
 import "./App.css";
 import { Route, Routes } from "react-router-dom";
 import { Home } from "./components/Home";
+import { ArticlePage } from "./components/ArticlePage";
 
 function App() {
   return (
     <>
       <Routes>
         <Route path="/home/articles" element={<Home />} />
+        <Route path="/home/article/:article_id" element={<ArticlePage />} />
```

Comment:
Is /home a good path name to have here as we are no longer on any sort of homepage?